### PR TITLE
Upgrade to version 1.6.0

### PR DESCRIPTION
**Pull request by HamedMasafi**

### CODE REVIEW

File: org.kde.kommit.json [hunk 1]
```diff
@@ -49,8 +49,8 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://download.kde.org/stable/kommit/kommit-1.0.2.tar.xz",
-                    "sha256": "1e8ecad23aa10508d0685a2e9169fc562e12fad5a6adaacd4542d9007a68e96e"
+                    "url": "https://download.kde.org/stable/kommit/kommit-1.6.0.tar.xz",
+                    "sha256": "ac30eb5f1a8c4035c6499d27325d499121ac2de7b3d381cacb0dd7427fb45025"
                 },
                 {
                     "type": "patch",
```
Comment (tsdgeos) [inline org.kde.kommit.json:67]: Seems like the patch either is not needed anymore or needs to be rebased

**Comment by HamedMasafi** [comment]
I checked; It doesn't needed anymore

File: org.kde.kommit.json [hunk 2]
```diff
@@ -40,6 +40,11 @@
                         "stable-only": true,
                         "url-template": "https://www.kernel.org/pub/software/scm/git/git-$version.tar.xz"
                     }
+                },
+                {
+                    "type": "archive",
```
Comment (travier) [inline org.kde.kommit.json:46]: You need to make this an independent entry to build it, not just add it as a source to an existing one.

Comment (HamedMasafi) [reply org.kde.kommit.json:46]: I don't understand, can you give an example? This app is depend on the libgit2 library

Comment (travier) [reply org.kde.kommit.json:46]: Right now you are adding a source to the list of sources for the `git` module: https://github.com/flathub/org.kde.kommit/pull/17/files#diff-2380fcadc398f6311b6738898a3cacbbc95ba696411424a5ec10215a1c7f31d3R21

So it's basically unused.

To add another module where you build the `libgit2` library, you need to add:
```
        {
            "name": "libgit2",
			... various build options here ...
            "sources": [
                {
                    "type": "archive",
                    "url": "https://github.com/libgit2/libgit2/archive/refs/tags/v1.8.1.tar.gz",
                    "sha256": "8c1eaf0cf07cba0e9021920bfba9502140220786ed5d8a8ec6c7ad9174522f8e",
                    "x-checker-data": {
                        "type": "anitya",
                        "project-id": FIXME,
                        "stable-only": true,
                        "url-template": "FIXME"
                    }
                }
            ]
        },
```

Comment (travier) [reply org.kde.kommit.json:46]: It's mostly a copy paste of https://github.com/flathub/org.gnome.gitg/blob/596c941472ede90bae33f464f625fa5dd041f536/org.gnome.gitg.json#L70..L90

**Pull request merged**